### ISSUE TITLE
Test for WCAG failure F26: Using a graphical symbol alone to convey information

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#183]
http://www.w3.org/TR/2014/NOTE-WCAG20-TECHS-20140311/F26
The objective of this technique is to show how using a graphical symbol to convey information can make content difficult to comprehend. A graphical symbol may be an image, an image of text or a pictorial or decorative character symbol (glyph) which imparts information nonverbally. Examples of graphical symbols include an image of a red circle with a line through it, a "smiley" face, or a glyph which represents a check mark, arrow, or other symbol but is not the character with that meaning. Assistive technology users may have difficulty determining the meaning of the graphical symbol. If a graphical symbol is used to convey information, provide an alternative using features of the technology or use a different mechanism that can be marked with an alternative to represent the graphical symbol. For example, an image with a text alternative can be used instead of the glyph. 
## Example

A shopping cart uses two simple glyphs to indicate whether an item is available for immediate shipment. A check mark indicates that the item is in stock and ready to ship. An "x" mark indicates that the item is currently on back order and not available for immediate shipment.
## Test procedure
- Examine the page for non-text marks that convey information.
- Check whether there are other means to determine the information conveyed by the non-text marks.
